File: src/Programmers/P1/P1_12948.java
package Programmers.P1;

public class P1_12948 {
    public static void main(String[] args) {
            System.out.println(solution("01033334444"));
    }

    public static String solution(String phone_number) {
        char[] array = phone_number.toCharArray();
        int length = array.length;
        for(int i = length-5;i>-1;i--){
            array[i]='*';
        }
        return new String(array);
    }
}

File: src/Programmers/P1/P1_12948.kt
package Programmers

fun main() {
    println(solution("01033334444"))
    println(solution("027778888"))
    println(solution("1234"))
}

private fun solution(phone_number: String): String {
    return (0..phone_number.lastIndex - 4).map { "*" }.joinToString("") + phone_number.substring(phone_number.lastIndex - 3)
}